app/client/Widgets/NothingPlayingWidget.cpp

```diff
@@ -110,6 +128,12 @@ NothingPlayingWidget::oniTunesClicked()
     startApp( "/iTunes/iTunes.exe" );
 }
 
+void
```

Comment:
[minor] This should be in the same line as the function name.

Reply:
I would choose to do that if this was a new file, but I think it makes more sense to follow the style already in the file.